showkase-processor-testing/src/test/resources/ShowkaseProcessorTest/top_level_composable_function_with_showkase_and_showkaseroot_with_tags_and_metadata/output/comairbnbandroidshowkaseprocessortestinggroupname.kt

```diff
@@ -0,0 +1,16 @@
+// This is an auto-generated file. Please do not edit/modify this file.
+package com.airbnb.android.showkase_processor_testing
+
+import androidx.compose.runtime.Composable
+import com.airbnb.android.showkase.models.ShowkaseBrowserComponent
+
+public val comairbnbandroidshowkaseprocessortestinggroupname: ShowkaseBrowserComponent =
+    ShowkaseBrowserComponent(
+        group = "group",
+        componentName = "name",
+        componentKDoc = "",
+        componentKey = """com.airbnb.android.showkase_processor_testing_null_group_name_0_null""",
+        isDefaultStyle = false,
+        tags = listOf("tag A", "tag B"),
```

Comment:
you want the extra metadata to be available here as well no?

Reply:
Oops I missed that the other classes were internal 🤦